patcher/Patcher.py

```
@@ -303,15 +287,29 @@ def scan_for_headers_in_section(sections_path: Path):
     return folders
 
 
-def run_system(command: str) -> int:
+def run_system(template: Template) -> int:
+    if not isinstance(template, Template):
+        raise Exception("Not a template")
```

Ruff (0.14.14)

Prefer `TypeError` exception for invalid type

(TRY004)